make_release/release-note/gh-release-excerpt

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env nu
+
+
+# Prepare the GitHub release text
+def main [
+    versionname: string # The version we release now
+    bloglink: string # The link to the blogpost
+    date?: datetime  # the date of the last release (default to 3 weeks ago, excluded)
+] {
+    let date = (
+        if $date == null { (date now) - 3wk + 1day } else { $date }
+        | date format "%Y-%m-%d"
+    )
```

Comment:
as in https://github.com/nushell/nu_scripts/pull/561, i would
- the `default` command
- switch to the 4-week schedule

:yum:

Reply:
Ah right I copied the code on main without looking to deep into that, and yesterday I was running it with explicit dates anyway.